File: ITManagementClient/Models/ResponseModels/Requests/GetRequestFullInfoResponseModel.cs
using ITManagementClient.Models.Enums;

namespace ITManagementClient.Models.ResponseModels.Requests
{
    public class GetRequestFullInfoResponseModel
    {
        public string RequestDescription { get; set; }

        public RequestType RequestType { get; set; }

        public RequestStatus RequestStatus { get; set; }

        public string ResolveNotes { get; set; }
    }
}

File: ITManagementServer/src/main/java/models/responseModels/requests/GetRequestFullInfoResponseModel.java
package models.responseModels.requests;

public class GetRequestFullInfoResponseModel {
    private String RequestDescription;
    private int RequestType;
    private int RequestStatus;
    private String ResolveNotes;

    public String getRequestDescription() {
        return RequestDescription;
    }

    public void setRequestDescription(String requestDescription) {
        RequestDescription = requestDescription;
    }

    public int getRequestType() {
        return RequestType;
    }

    public void setRequestType(int requestType) {
        RequestType = requestType;
    }

    public int getRequestStatus() {
        return RequestStatus;
    }

    public void setRequestStatus(int requestStatus) {
        RequestStatus = requestStatus;
    }

    public String getResolveNotes() {
        return ResolveNotes;
    }

    public void setResolveNotes(String resolveNotes) {
        ResolveNotes = resolveNotes;
    }
}